include/ccl.h

```diff
@@ -20,21 +20,17 @@ extern "C" {
  *For the latest version of CCL, please visit the github repository: https://github.com/LSSTDESC/CCL
  */
 
-#pragma once
-
+#include "ccl_defs.h"
+#include "ccl_utils.h"
+#include "ccl_config.h"
 #include "ccl_core.h"
+#include "ccl_error.h"
 #include "ccl_constants.h"
 #include "ccl_power.h"
 #include "ccl_cls.h"
-#include "ccl_utils.h"
 #include "ccl_background.h"
-#include "ccl_config.h"
 #include "ccl_correlation.h"
 #include "ccl_massfunc.h"
 #include "ccl_neutrinos.h"
-  
-#include "ccl_placeholder.h"
 
-#ifdef __cplusplus
-}
 #endif
```

Comment:
Some people add `/*  __CCL_H_INCLUDED__ */` after the `#endif` to help make the code more readable. I am 50/50 on it but it can be nice.

Reply:
I agree usually conditional compilations the matched end is good. In this case though, an important guard protected file must end with an endif, so it has to be the first ifdef at the top of file; there is no confusion. Plus it is less likely to miss change the name when the only two occurrences are right next to each other.